Xamarin.Forms.Core/IndicatorView.cs

```diff
@@ -106,18 +106,22 @@ public IEnumerable ItemsSource
 			set => SetValue(ItemsSourceProperty, value);
 		}
 
-		public static readonly BindableProperty ItemsSourceByProperty = BindableProperty.Create("ItemsSourceBy", typeof(VisualElement), typeof(IndicatorView), default(VisualElement), propertyChanged: (bindable, oldValue, newValue)
-		 => LinkToCarouselView(bindable as IndicatorView, newValue as CarouselView));
 
 		[TypeConverter(typeof(ReferenceTypeConverter))]
-		public static VisualElement GetItemsSourceBy(BindableObject bindable)
+		public VisualElement ItemsSourceBy
```

Comment:
if we go with this, we need a better name

Reply:
the property accepts any `VisualElement` but the app will do nothing unless it's a CarouselView ?

Reply:
Only takes ItemsView now, just works if it's a carousel but user can override the virtual method for other controls